pysoa/common/transport/base.py

```diff
@@ -119,8 +119,18 @@ def receive_response_message(self, receive_timeout_in_seconds=None):
 class ServerTransport(Transport):
     """
     The base server transport defining the interface for transacting PySOA payloads on the server side.
+
+    :param service_name: The name of the service for which this transport will receive requests and send responses
+    :param instance_index: The 1-based index of this process among multiple forks
+    :param metrics: The optional metrics recorder
     """
 
+    def __init__(self, service_name, instance_index, metrics=noop_metrics):
```

Comment:
This is technically a breaking change to add a new required positional argument. We'll want to make sure we note this in the changelog for others that have created their own `ServerTransport` subclasses.